### ISSUE TITLE
`deno bundle` does not bundle imports with @deno-types annotation

### ISSUE THREAD

**Issue by nitsky** [denoland/deno#6338]
Hi, I am observing that `deno bundle` does not bundle imports that have the `@deno-types` annotation. This may be related to #5665.

```
// @deno-types="https://unpkg.com/rollup@2.16.0/dist/rollup.d.ts"
import { rollup } from 'https://unpkg.com/rollup@2.16.0/dist/es/rollup.browser.js'
console.log(rollup)
```

`deno bundle test.js > out.js && deno run out.js`

```
error: Uncaught TypeError: Cannot read property 'rollup' of undefined
            console.log(rollup_browser_js_1.rollup);
```

If I omit the `@deno-types` annotation, the `out.js` file is significantly larger and it works fine.

**Comment by kitsonk** [comment]
Actually this is a duplicate of #4539, just it is also `@deno-types` as well.

**Comment by kitsonk** [comment]
This was resolved in 1.5 and the way we handle the modules and bundling.  The example no longer produces the reported error.
